src/Features/Core/Portable/ExtractMethod/MethodExtractor.VariableSymbol.cs

```diff
@@ -75,6 +72,23 @@ protected VariableSymbol(Compilation compilation, ITypeSymbol type)
 
                 return left.DisplayOrder - right.DisplayOrder;
             }
+
+            private static bool IsCancellationToken(ITypeSymbol originalType)
+            {
+                return originalType is
+                {
+                    Name: nameof(CancellationToken),
+                    ContainingNamespace:
+                    {
+                        Name: nameof(System.Threading),
+                        ContainingNamespace:
+                        {
+                            Name: nameof(System),
```

Comment:
>Name: nameof(System), [](http://example.com/codeflow?start=28&length=21)

The System.Threading nameof check above isn't sufficient?

Reply:
Just being thorough:-)